cloud/amazon/ec2_elb_lb.py

```diff
@@ -858,6 +868,23 @@ def main():
     if state == 'present' and not (zones or subnets):
         module.fail_json(msg="At least one availability zone or subnet is required for ELB creation")
 
+    if security_group_ids and security_group_names:
```

Comment:
Would be preferred if `mutually exclusive` is leveraged within the argument spec object.